<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWidgetClass</class>
 <widget class="QWidget" name="VideoWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>图像</string>
     </property>
     <layout class="QHBoxLayout" name="HLayout_VideoFrame"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>240</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>导入信息</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_file_FileName">
           <property name="text">
            <string>文件名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_file_Duration">
           <property name="text">
            <string>持续时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_codec">
           <property name="text">
            <string>视频编码器：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_resolution">
           <property name="text">
            <string>加载分辨率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_frameRate">
           <property name="text">
            <string>帧率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_BitRate">
           <property name="text">
            <string>视频比特率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_pixFmt">
           <property name="text">
            <string>像素格式：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_colorType">
           <property name="text">
            <string>颜色格式：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>导入帧整体信息</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_totalFrameCount">
             <property name="text">
              <string>总帧数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_FrameCount_I">
             <property name="text">
              <string>I帧数量：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_FrameCount_P">
             <property name="text">
              <string>P帧数量：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_FrameCount_B">
             <property name="text">
              <string>B帧数量：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>当前帧信息</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_currentFrameIndex">
             <property name="text">
              <string>帧索引：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_currentFrameType">
             <property name="text">
              <string>帧类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_currentFrameTime">
             <property name="text">
              <string>帧所在时间：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_currentFrameDuartion">
             <property name="text">
              <string>帧持续时间：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_currentFrameTime_2">
            <property name="text">
             <string>当前时间:00:00:00.000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="HSlider_currentFrameIndex">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_play">
            <property name="text">
             <string>play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pause">
            <property name="text">
             <string>pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_stop">
            <property name="text">
             <string>stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_nextFrame">
            <property name="text">
             <string>下一帧</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_nextIFrame">
            <property name="text">
             <string>下一个I帧</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ExtraWindow">
            <property name="text">
             <string>弹窗</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
